src/frontend/komponenter/Felleskomponenter/Hendelsesoversikt/Header/Header.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { useBehandling } from '../../../../context/BehandlingContext';
+import Dokumenterknapp from './Dokumenterknapp';
+import Historikkknapp from './Historikkknapp';
+import Meldingerknapp from './Meldingerknapp';
+import { Tabs } from '../typer';
+
+interface IProps {
+    aktivTab: Tabs;
+    settAktivTab: (tab: Tabs) => void;
+}
+
+const StyledHeader = styled.div`
+    height: 4rem;
+    padding: 0 1.25rem;
+    display: flex;
+`;
+
+const Header = ({ aktivTab, settAktivTab }: IProps) => {
+    const { erLesevisning } = useBehandling();
+
+    return (
+        <StyledHeader>
+            <Historikkknapp
```

Comment:
Allerede en del endringer her, så kan gjøres i egen pr, men vi bør ta en runde på om vi vil kalle ting for historikk eller hendelser og meldinger eller brev kanskje :)

Reply:
Helt enig. Backend kaller vi jo dette for logg, så er litt ulikt frontend og backend også.